client/lib/abtest/index.js

```diff
@@ -87,6 +88,13 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
+	const languageSlugs = map( config( 'languages' ), 'langSlug' );
+	const localeTargets = [ 'not-en', 'any' ].concat( languageSlugs );
+
+	if ( testConfig.localeTargets && ! includes( localeTargets, testConfig.localeTargets ) ) {
+		throw new Error( 'localeTargets can by "any", "not-en" or any single locale slug.' );
```

Comment:
Not naive, but I'm just trying to keep it simple for anyone who's going to use this, and try not to have to add code to handle human errors there.